local/bin/py/build/configurations/pull_config_preview.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```

Comment:
This is a global file change, just want to confirm that this is intentional.

Reply:
Thanks! Yeah, we temporarily change this for the preview to work, but I'll change it back before merging.

Reply:
Yep, this was just a temporary change for the preview to work correctly. I reverted it back now. 👍